### PR TITLE
Add script to null values of minimal images to force rebuild

### PR DESCRIPTION
**Pull request by rcrozean**
*Issue #, if available:*

*Description of changes:*
Add the script and make value to null all values in EKS_DISTRO_TAG_FILE.yaml. If this is run and the results pushed, it would force a rebuild of the minimal images. Will add a periodic prow job to trigger this every quarter.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pr-scripts/create_pr.sh [hunk 1]
```diff
@@ -42,6 +42,10 @@ if [[ $JOB_NAME =~ "prow-deck-tooling" ]]; then
     CHANGED_FILE="Prow controlplane Helm chart"
     CHANGED_COMPONENT="Prow component images"
 fi
+if [[ $JOB_NAME =~ "quarterly-minimal-image-rebuild" ]]; then
```
Comment (danbudris) [inline pr-scripts/create_pr.sh:45]: set job name as a var and re-use in multiple places so we can change it once if we need to?

File: eks-distro-base/minimal-image-rebuilder.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+set -e
```
Comment (danbudris) [inline eks-distro-base/minimal-image-rebuilder.sh:1]: add the bash shebang and copyright disclaimer? See https://github.com/aws/eks-distro-build-tooling/blob/main/pr-scripts/create_pr.sh#L1

File: pr-scripts/rebuild-minimal-images-pr-body [hunk 3]
```diff
@@ -0,0 +1 @@
+This PR updates all tags in EKS_DISTRO_TAG_FILE to `null`. Merging this PR will cause all minimal images to be rebuilt. Before approving this PR, check when the last rebuild occurred by checking the history of [EKS_DISTRO_TAG_FILE](https://github.com/aws/eks-distro-build-tooling/EKS_DISTRO_TAG_FILE.yaml)
```
Comment (danbudris) [inline pr-scripts/rebuild-minimal-images-pr-body:1]: Should we add some emoji? 😸

File: eks-distro-base/Makefile [hunk 4]
```diff
@@ -776,6 +780,9 @@ update-base-image-other-repos:
 .PHONY: all
 all: release
 
+.PHONY: rebuild-minimal-images
```
Comment (danbudris) [inline eks-distro-base/Makefile:783]: let's add docstring-like comments to these to document what they're for, one sentance is fine

File: eks-distro-base/minimal-image-rebuilder.sh (hunk 2 above)
Comment (danbudris) [inline eks-distro-base/minimal-image-rebuilder.sh:1]: let's also add a comment to the top of this which talks about what it's for and why, for posterity sake

File: pr-scripts/rebuild-minimal-images-pr-body (hunk 3 above)
Comment (rcrozean) [reply pr-scripts/rebuild-minimal-images-pr-body]: Went with a fun chatops comment provided by prow.

File: eks-distro-base/minimal-image-rebuilder.sh (hunk 2 above)
Comment (rcrozean) [reply eks-distro-base/minimal-image-rebuilder.sh:23]: Done.

**Pull request merged**